_episodes/02-variables.md

```diff
@@ -165,51 +165,6 @@ num_governors = governors_per_state * num_states
     need to change the number once.
 *   The same applies for other data types - avoid "magic strings" as well
 
-
-## Use an index to get a single character from a string.
```

Comment:
since you've removed a lot here, can you update the teaching/exercise time at the top of the page?